### PR TITLE
Release 0.5.1

### PR DESCRIPTION
**Pull request by blootsvoets**
radar-commons-android version 0.5.1

Changes since versions 0.5:
- Wait 60 seconds before retrying managementportal connection
- Setup logging and crash handling with Crashlytics
- Added authentication token tests

### CODE REVIEW

File: src/main/java/org/radarcns/android/device/DeviceService.java [hunk 1]
```diff
@@ -559,20 +558,23 @@ public void registerDevice(String sourceIdHint, String name, Map<String, String>
...
-                            // TODO: more error handling?
-                            logger.error("Failed to register source {}", source);
-                            stopDeviceManager(unsetDeviceManager());
+                            // try again in a minute
+                            handler.postDelayed(new Runnable() {
+                                @Override
+                                public void run() {
+                                    stopDeviceManager(unsetDeviceManager());
+                                }
+                            }, 60_000L);
```
Comment (strangepleasures) [inline src/main/java/org/radarcns/android/device/DeviceService.java]: It would probably make sense to use a random delay

File: .travis.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@ android:
     - tools
     - build-tools-26.0.1
```
Comment (strangepleasures) [inline .travis.yml]: Can it probably be removed?

Comment (blootsvoets) [reply .travis.yml]: Unfortunately it needs both, or it starts complaining about missing licenses...

**Comment by blootsvoets** [comment]
@strangepleasures could you take another look?

**Pull request merged**